bats/tests/containers/published-ports.bats

```diff
@@ -0,0 +1,45 @@
+load '../helpers/load'
+
+skip_unless_host_ip() {
```

Comment:
Should this start going into a helper file? That's the second time I've noticed this.

Reply:
Sounds good, but I'm still not sure if `HOST_IP` is really the right name, given that we set it to the `vEthernet (WSL)` address on Windows, which isn't really the host address. I've discussed this with @Nino-K before, and we agreed to use it for now, but I'm still not quite happy with it.